configs/pcx86/machine/compaq/lte286/cga/debugger/machine.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="/configs/pcx86/xsl/machine.xsl"?>
+<machine id="compaq-lte286-640k" type="pcx86" border="1" pos="center" background="default">
+	<name pos="center">COMPAQ LTE/286, 640Kb RAM</name>
+	<computer id="compaq-lte286-640k" name="COMPAQ LTE/286" buswidth="24"/>
+	<cpu id="cpu286" model="80286" cycles="12000000"/>
+	<ram id="ramLow" addr="0x00000" size="0xa0000"/>
+	<rom id="romBIOS" addr="0xf0000" size="0x10000" alias="0xff0000" file="/machines/pcx86/compaq/lte286/rom/1989-09-07/1989-09-07.json"/>
+	<video ref="/configs/pcx86/video/ibm/cga/ibm-cga.xml"/>
+	<fdc ref="/configs/pcx86/fdc/library.xml" autoMount='{A: {name: "SystemPro EISA Configuration 1.15"}}'/>
+	<!-- TODO: <hdc ref="/configs/pcx86/hdc/20mb/unformatted-at2.xml"/> -->
+	<keyboard ref="/configs/pcx86/keyboard/us84-softkeys.xml"/>
+        <debugger id="debugger" messages="" commands=""/>
```

Comment:
Is the difference in indentation here just a tabs-vs-space issue?

Reply:
> Is the difference in indentation here just a tabs-vs-space issue?

@jeffpar I believe my editor was using its defaults and put in real tab characters.

Do you have a javascript code beautifier? I use clang-format at work for C